app/server/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
     "compilerOptions": {
+        "target": "es2019",
```

Comment:
Needed to add this to allow jest to run tests of code using null-coalescing operator

Reply:
we can presumably target whatever we want, no need to avoid new features